setup.py

```diff
@@ -29,31 +30,33 @@
 Alternatively, you can install from source with
 
     ./setup.py install --user
-"""
 
+"""
 
-from __future__ import print_function
 
 import sys
+
+from io import open
 from warnings import warn
 
 from setuptools import setup
 
 
-def get_version():
-    """Load the version from version.py, without importing it.
+def version():
```

Comment:
Made this read version string from final line of \_\_init\_\_.py so that users can install via setup.py.

Reply:
"users" being my scrum group :)